### PR TITLE
HZC-2739 spring boot sample application to establish connection with hazelcast in cloud

### PR DESCRIPTION
**Pull request by ivan-shevtsov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,6 @@ You will need at least a *free* account on [Hazelcast Cloud](https://cloud.hazel
         
 - <h3><a href="https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/serverless-fraud-detection">serverless-fraud-detection</a></h3>
         Serverless fraud detection, using Node.js and Amazon Lambda        
+
+  <h3><a href="https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/spring-sample">spring-sample</a></h3>
```
Comment (JakeSCahill) [inline README.md:18]: We should make sure to use Hazelcast Viridian instead of Hazelcast Cloud in all READMEs.

File: spring-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#Spring Boot Application with Hazelcast Cloud Cluster
```
Comment (JakeSCahill) [inline spring-sample/README.md:1]:
```suggestion
# Spring Boot Application with Hazelcast Viridian Cluster
```

Replace Cloud with Viridian in the rest of the doc.

**Review by JakeSCahill — APPROVED** [review]
It's working 😄 We should just edit the README to use the new naming.

File: README.md (hunk 1 above)
Comment (ivan-shevtsov) [reply README.md:18]: only in text messages or even in application properties?

Comment (JakeSCahill) [reply README.md:18]: Will it work if we rename those?

File: spring-sample/README.md (hunk 2 above)
Comment (ivan-shevtsov) [reply spring-sample/README.md]: renamed

File: README.md (hunk 1 above)
Comment (ivan-shevtsov) [reply README.md:18]: done

File: spring-sample/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+hazelcast.viridian.url=https://dev.test.hazelcast.cloud
```
Comment (JakeSCahill) [inline spring-sample/src/main/resources/application.properties:1]: Will we need the URL for production? Doesn't the client default to the production cloud environment?

Comment (kubamarchwicki) [reply spring-sample/src/main/resources/application.properties:1]: that would be `coordinator.hazelcast.com` - which would be different again on the new environment 
Yes. the client defaults to production environment

Comment (JakeSCahill) [reply spring-sample/src/main/resources/application.properties:1]: I think to avoid confusing customers/users, we should remove this.

**Comment by JakeSCahill** [comment]
Do we need to duplicate the Maven Wrapper parts? The repo already has the wrapper at its root.

**Comment by ivan-shevtsov** [comment]
> 

this is not a submodule, so it is better to use independent maven wrapper for it

**Pull request merged**